### PR TITLE
Fix workspace loading issue

### PR DESCRIPTION
**Pull request by akoptelov**
This fixes issue #31.

### CODE REVIEW

**Comment by reinhapa** [comment]
Please clean up PR to only include the actual changes and not just formatting changes. I will check to update the README what the correct formatting rules must be.

File: rtc2git.cli.extension/src/to/rtc/cli/migrate/RtcMigrator.java [hunk 1]
```diff
@@ -138,7 +138,7 @@ private void handleInitialLoad(RtcChangeSet changeSet) {
 				initiallyLoadedComponents.add(changeSet.getComponent());
 			} catch (CLIClientException e) { // ignore
 				throw new RuntimeException("Not a valid sandbox. Please run [scm load " + workspace
-						+ "] before [scm migrate-to-git] command", e);
```
Comment (reinhapa) [inline rtc2git.cli.extension/src/to/rtc/cli/migrate/RtcMigrator.java]: You have accidentally removed the passing of the causing exception here

**Comment by akoptelov** [comment]
@reinhapa I'd gladly do. Still, the project has format-on-save and organize imports options turned on without any specific formatting, so this is the result of Eclipse default formatting. In this circumstances it is quite problematic to keep formatting while working on something bigger than this one.

**Comment by reinhapa** [comment]
@akoptelov  I have to check with what the actual formatting rules should be and then reformat the complete project later

**Comment by akoptelov** [comment]
@reinhapa Patrik, please hold on with merging. It looks like my fix breaks migration of components started with initial baseline. I'll try to figure out why.

**Comment by akoptelov** [comment]
@reinhapa Sorry for panicking, now it works OK for components with both initial and non-initial baselines. I only did a little fix to remove "force" component loading in case of update mode (now it should't be needed as we track components). So please merge it as soon as your review it.

**Review by reinhapa — APPROVED** [review]

**Pull request merged**